corehq/apps/case_importer/base.py

```diff
@@ -19,10 +20,14 @@ def location_safe_case_imports_enabled(view_func, request, *args, **kwargs):
 class ImportCases(DataInterface):
     name = gettext_lazy("Import Cases from Excel")
     slug = "import_cases"
-    report_template_path = "case_importer/bootstrap3/import_cases.html"
+    report_template_path = "case_importer/bootstrap5/import_cases.html"
     hide_filters = True
     asynchronous = False
 
+    def __init__(self, request, **kwargs):
+        super(ImportCases, self).__init__(request, **kwargs)
+        set_bootstrap_version5()
```

Comment:
Hi Jenny, I'm reviewing your PR and don't understand this part, why and when will we need to call `set_bootstrap_version5()`?

Reply:
I'm doing this because the `@use_bootstrap5` decorator doesn't work here.

`ImportCases` isn't a traditional view with a `dispatch` method. It's a `DataInterface` which is a `genericReportView` which inherits from `object` and dispatches views via a `ReportDispatcher` attribute.

`set_bootstrap_verstion5` is all that `@use_bootstrap5` does, check it out [here](https://github.com/dimagi/commcare-hq/blob/4f0cc77b232fec5209ee24fac829704b6e703c78/corehq/apps/hqwebapp/decorators.py#L232).